pom.xml

```diff
@@ -2100,7 +2100,7 @@
                 <!-- marked provided due to license issues -->
                 <groupId>jline</groupId>
                 <artifactId>jline</artifactId>
-                <version>1.0</version>
+                <version>2.11</version>
```

Comment:
Has build-cqs.sh been re-run?  I don't see any changes related to jline.

Reply:
it has been - since jline is a provided scope and not directly imported by us (i.e. it's only a transitive dep), it doesn't show up in the cqs list.